grakn-dist/src/grakn

```diff
@@ -0,0 +1,662 @@
+    fi
+  else
+    echo "NOT RUNNING"
+  fi
+
+  return $status
+}
+
+queue_clean_queue() {
+  local queue_bin=
+  if [ "$(uname)" == "Darwin" ]; then
+      queue_bin="redis-cli-osx"
+  elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
+      queue_bin="redis-cli-linux"
+  fi
+
+  "${GRAKN_HOME}/services/redis/"$queue_bin flushall
+}
+
+# ================================================
+# grakn helper functions
+# ================================================
+grakn_start_grakn() {
+  local status=
+  java -cp "${CLASSPATH}" -Dgrakn.dir="${GRAKN_HOME}/services" -Dgrakn.conf="${GRAKN_CONFIG}" \
+    ai.grakn.engine.GraknEngineServer > /dev/null 2>&1 &
+  status=$?
+  pid=`ps -ef | grep GraknEngineServer | grep -v grep | awk '{ print $2}'`
+  echo $pid > "${GRAKN_PID}"
+  return $status
+}
+
+grakn_is_grakn_initiated() {
+  if [ -e "$GRAKN_PID" ] && ps -p `cat $GRAKN_PID` > /dev/null ; then
+    return 0
+  else
+    return 1
+  fi
+}
+
+grakn_is_grakn_ready() {
+  java -cp "${CLASSPATH}" -Dgrakn.dir="${GRAKN_HOME}/services" -Dgrakn.conf="${GRAKN_CONFIG}" ai.grakn.client.Client > /dev/null 2>&1
+  return $?
+}
+
+grakn_wait_until_ready() {
+  local now_s=`date '+%s'`
+  local stop_s=$(( $now_s + $GRAKN_STARTUP_TIMEOUT_S ))
+
+  while [ $now_s -le $stop_s ]; do
+      echo -n "." # output some sort of "progress bar"
+      grakn_is_grakn_ready
+      local is_grakn_running=$?
+      if [ $is_grakn_running -eq 0 ] ; then
+          return 0
+      fi
+      sleep $WAIT_INTERVAL_S
+      now_s=`date '+%s'`
+  done
+
+  return 1
+}
+
+grakn_start_and_wait_until_ready() {
+  grakn_is_grakn_initiated
+  is_grakn_running=$?
+
+  if [ $is_grakn_running -eq 0 ]; then
+    echo "Grakn is already running"
+    return 0 # grakn is already running. no need to do anything
+  else
+    # otherwise attempt to start
+    echo -n "Starting Grakn..."
+    grakn_start_grakn
+    local exec_grakn_bin=$?
+
+    # check grakn status and wait until ready
+    local status=
+    if [ $exec_grakn_bin -eq 0 ] ; then
+      # exec grakn bin successful. now wait until ready
+      grakn_wait_until_ready
+      status=$?
+      echo "SUCCESS"
+    else
+      # exec grakn bin failed
+      status=1
+      echo "FAILED!"
+    fi
+
+    return $status
+  fi
+}
+
+grakn_server_stop_grakn() {
+  echo -n "Stopping Grakn..."
+  grakn_is_grakn_initiated
+  local is_grakn_running=$?
+  if [[ $is_grakn_running -eq 0 ]]; then
+    if [[ -e "$GRAKN_PID" ]]; then
+      kill `cat $GRAKN_PID`
+      status_kill=$?
+      rm $GRAKN_PID
+      if [ $status_kill -eq 0 ] ; then
+        echo "SUCCESS"
+      else
+        echo "FAILED!" # unable to kill
+      fi
+      return $status_kill
+    else
+      echo "FAILED!" # unable to find PID file
+      return 1
+    fi
+  else
+    echo "NOT RUNNING"
+  fi
+}
+
+grakn_clean_grakn() {
+  # redirect to /dev/null to silence 'No such file' error, when the directory is already empty
+  rm "${GRAKN_HOME}"/logs/*.log > /dev/null 2>&1
```

Comment:
just use `rm -f`

Reply:
fixed in ab34fb04dbe737d186426d757d291e7fd3707f7c